pyvista/core/filters.py

```diff
@@ -130,13 +130,13 @@ def clip_box(dataset, bounds=None, invert=True, factor=0.35):
 
         If no bounds are given, a corner of the dataset bounds will be removed.
 
-        Parameters
+        Parametersequence
```

Comment:
```suggestion
        Parameters
```

Reply:
Ugh, thanks for catching that, I have no idea how that happened :disappointed: